hooks/publish_extra_details.py

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) 2015 Shotgun Software Inc.
+#
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Shotgun Software Inc.
+
+from sgtk import Hook
+
+
+class PublishExtraDetails(Hook):
+    """Hook that can be used to add extra fields in the details pane."""
+
+    def execute(self, sg_item, **kwargs):
```

Comment:
If you're not passing the status model method in (see later comment below) then you can remove the `**kwargs` here. We do use `**kwargs` in some hooks and elsewhere when needed, but if there's not a specific need for it then it's best to stick with a strict signature.

Reply:
I think that **kwargs gives flexibility for implementing any functionality inside the hook